ananas/src/main/java/iamutkarshtiwari/github/io/ananas/editimage/fragment/TextEditorDialogFragment.java

```diff
@@ -52,7 +52,7 @@ public static TextEditorDialogFragment show(@NonNull AppCompatActivity appCompat
 
     //Show dialog with default text input as empty and text color white
     public static TextEditorDialogFragment show(@NonNull AppCompatActivity appCompatActivity) {
-        return show(appCompatActivity, "", ContextCompat.getColor(appCompatActivity, R.color.white));
+        return show(appCompatActivity, "", ContextCompat.getColor(appCompatActivity, R.color.black));
```

Comment:
should this be themed (dark/light) or is the image editor only opening on one theme?

Reply:
This is fine. It's not about the theme here. It's just used to show the default text colour in the editor